cypress/e2e/apps.js

```diff
@@ -146,6 +146,7 @@ describe("As a staff user I want to manage apps", () => {
         .click()
         .get(BUTTON_SELECTORS.confirm)
         .click()
+        .wait(3000)
```

Comment:
I thought that we've agreed to not add explicit waits and instead we should wait for a specific action/element.

Reply:
yes but in this case, we want to make it work. It is Cypress that we are removing, it is a temporary quick solution